### ISSUE TITLE
Sometimes Thorium doesnt remember URL changes, for back & forward history buttons.

### ISSUE THREAD

**Issue by Username23453** [Alex313031/thorium#234]
Sometimes this rarely happens, For example when i go to tiktok: https://www.tiktok.com/@splashpiggy/video/7257642085837442350

THEN press the down directional arrow on my keyboard. The down arrow will skip to the next video AND the URL will change. But when you go back/press history back button, it doesnt remember the @splashpiggy/video/7257642085837442350 URL for some reason, which is odd.

This is also a security danger. If you are using a website & an aggressive popup hijacks your URL & changes it. You need to be able to press the back button & return to the original authentic site you were on, whether that be google, or some other site.

**Comment by gz83** [comment]
If Chrome, Firefox, and Chromium are all like this, then it’s the website provider’s problem.

@Username23453

**Comment by gz83** [comment]
I can understand your concerns, but if this is the case with several other browsers, then Thorium should not make changes casually, because this may change the user's experience and behavior.

@Username23453

**Comment by gz83** [comment]
1. This is a batch closing operation (can also be regarded as a ping), I will reopen the issues that still have new responses

**Comment by gz83** [comment]
Does version 150 still have this issue? Assuming you are still using Thorium.

**Comment by Username23453** [comment]
Yes version 150 still has the issue, you can test if for yourself, see my original post here above.
